src/app/views/sessions/signin/true-command-status/true-command-status.component.html

```diff
@@ -0,0 +1,8 @@
+<div *ngIf="connectionState?.connected" class="truecommand" (click)="goToTrueCommand()">
+  <img
+    class="truecommand-logo"
+    src="../../../../../assets/images/truecommand/truecommand-logo-mark-full-color-rgb.svg"
+    alt="TrueCommand"
+  >
```

Comment:
let's self close the tag

Reply:
Closing tags like this was necessary in xhtml days and not necessary for html5.

Reply:
well, I'm old fashioned man then :D